src/org/jitsi/impl/neomedia/rtp/RTPEncodingImpl.java

```diff
@@ -279,11 +279,15 @@ public boolean matches(byte[] buf, int off, int len)
             return false;
         }
 
+        if (temporalId == -1)
+        {
+            return true;
+        }
+
         int tid = track.getMediaStreamTrackReceiver()
             .getStream().getTemporalID(buf, off, len);
 
-        return (tid == -1 && ArrayUtils.isNullOrEmpty(dependencyEncodings))
-                || tid == temporalId;
+        return tid == temporalId;
```

Comment:
I know you are not suggesting they are the same, and I'm sorry if it came out that way, or if it sounded harsh. But you are suggesting that we assume something about TS if simulcast is in use, which means that we handle them as if they are not independent. And we could easily not make the extra assumption, and support both cases without any signalling.